src/main/java/jp/co/soramitsu/sora/crypto/algorithms/SignatureSuiteRegistry.java

```diff
@@ -0,0 +1,28 @@
+package jp.co.soramitsu.sora.crypto.algorithms;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SignatureSuiteRegistry {
+
+  private static Map<String, RawSignatureStrategy> algorithms = new HashMap<>();
```

Comment:
will we call methods of this class in multithread environment? if answer Yes, then we should change HashMap to ConcurrentHashMap

Reply:
This map is created statically at compile time and never changed at run-time.